### PR TITLE
WEBUI-323: Remove temporary picker page provider definition (3.0.x)

### PR DESCRIPTION
**Pull request by mnixo**
Backport of #1213.

### CODE REVIEW

**Review by semisse — APPROVED** [review]

**Comment by mnixo** [comment]
Still requires referencing the appropriate platform version containing the final page provider definition.

**Review by richardsd — APPROVED** [review]

**Review by richardsd — DISMISSED** [review]

**Review by richardsd — DISMISSED** [review]

**Review by semisse — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

File: plugin/web-ui/marketplace/src/main/resources/package.xml [hunk 1]
```diff
@@ -11,9 +11,7 @@
   <nuxeo-validation>nuxeo_certified</nuxeo-validation>
   <production-state>production_ready</production-state>
   <supported>true</supported>
-  <platforms>
-    <platform>@DISTRIBUTION_NAME@-@NUXEO_TARGET_VERSION@</platform>
-  </platforms>
+  <target-platform>[2021.2.43,)</target-platform>
```
Comment (Gabez0r) [inline plugin/web-ui/marketplace/src/main/resources/package.xml:14]: I assume specifying the `name` is optional?

(question base on nuxeo-arender:
https://github.com/nuxeo/nuxeo-arender-connector/blob/2021.1/packages/nuxeo-arender-package/src/main/resources/package.xml#L14-L15)

**Comment by mnixo** [comment]
🔵  https://qa.nuxeo.org/jenkins/job/2021/job/web-ui-pipeline/63

**Review by troger — APPROVED** [review]

**Review by richardsd — APPROVED** [review]

**Comment by mnixo** [comment]
🔵  https://qa.nuxeo.org/jenkins/job/2021/job/web-ui-pipeline/65/

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**